docs/src/content/docs/ref/core-advanced.mdx

```diff
@@ -29,6 +171,110 @@ Usage example:
+
+However, regardless of the values of `cells` and `bits`, this function always adds the minimum price based on the value of [`lump_price`][param-24-25]. See the example for [`getSimpleForwardFee(){:tact}`](#getsimpleforwardfee) to derive it.
+
+Attempts to specify negative number of `cells` or `bits` throw an exception with [exit code 5](/book/exit-codes#5): `Integer out of expected range`.
+
+Usage example:
+
+```tact
+let fee: Int = getForwardFee(1_000, 1_000, false);
+//                           -----  -----  -----
+//                           ↑      ↑      ↑
+//                           |      |      Both source and destination
+//                           |      |      isn't on the masterchain,
+//                           |      |      but on the basechain
+//                           |      Number of bits in a message
+//                           Number of cells in a message
+```
+
+:::note[Useful links:]
+
+  [Forward fee in TON Docs][forward-fee]\
+  [Forward fee calculation in TON Docs][forward-fee-calc]\
+  [`getSimpleForwardFee(){:tact}`](#getsimpleforwardfee)\
+  [`getOriginalFwdFee(){:tact}`](#getoriginalfwdfee)
+
+:::
+
+## getSimpleForwardFee
+
+<Badge text="Available since Tact 1.5" variant="tip" size="medium"/><p/>
+
+```tact
+fun getSimpleForwardFee(cells: Int, bits: Int, isMasterchain: Bool): Int;
+```
+
+Similar to [`getForwardFee(){:tact}`](#getforwardfee), but without the [`lump_price`][param-24-25], i.e. without the minimum price to pay regardless of the amount of `cells` or `bits`. Calculates and returns only the `cells` times the current cell price plus `bits` times the current bit price.
+
+Attempts to specify negative number of `cells` or `bits` throw an exception with [exit code 5](/book/exit-codes#5): `Integer out of expected range`.
+
+Usage example:
+
+```tact
+let fee = getForwardFee(1_000, 1_000, false);
+let feeNoLump = getSimpleForwardFee(1_000, 1_000, false);
+let lumpPrice = fee - feeNoLump;
+```
+
+:::note[Useful links:]
+
+  [Forward fee in TON Docs][forward-fee]\
+  [Forward fee calculation in TON Docs][forward-fee-calc]\
+  [`getForwardFee(){:tact}`](#getforwardfee)
+
+:::
+
+## getOriginalFwdFee
+
+<Badge text="Available since Tact 1.5" variant="tip" size="medium"/><p/>
+
+```tact
+fun getOriginalFwdFee(fwdFee: Int, isMasterchain: Bool): Int;
+```
+
+Calculates and returns the so-called _original_ [forward fee][forward-fee] in [nanoToncoins](/book/integers#nanotoncoin) [`Int{:tact}`][int] for an outgoing message based on the `fwdFee` obtained from the incoming message. If both the source and the destination addresses are in the [basechain][basechain], then specify `isMasterchain` as `false{:tact}`. Otherwise, specify `true{:tact}`.
+
+This function is useful when the outgoing message depends heavily on the structure of the incoming message, so much so that you cannot fully predict the fee using [`getForwardFee(){:tact}`](#getforwardfee) alone.
```

Comment:
The official documentation says the same thing, but as far as I know the point is that such an instruction costs only 26 gas, as does GETFORWARDFEE, but GETFORWARDFEE requires a number of cells, and bits, for accurate counting, and CDATASIZE is not cheap. So if you need an approximate value rather than a nanotone precision, and the message contains almost the same as the incoming message, GETORIGINALFORWARDFEE is a good way to go